File: java/act3/act3a.java
package act3;

    import java.util.Scanner;

    public class act3a
     {
        public static void main(String[] args){
            System.out.println("\t############################################");
            System.out.printf("\t\t%5sGRADES CALCULATOR \n", "");
            System.out.println("\t############################################");

            // scanner instance
            Scanner scan = new Scanner(System.in);

            /*User Information*/
            System.out.print("\t\t  please enter your name: ");
            String name = scan.nextLine();

            System.out.print("\t\t  please enter your course: ");
            String course = scan.nextLine();

            /*Grades on quiz */
            System.out.print("\t\t  please enter your grade in quiz 1: ");
            double quiz1 = scan.nextInt();

            System.out.print("\t\t  please enter your grade in quiz 2: ");
            double quiz2 = scan.nextInt();
            
            /*Grades on recitation */
            System.out.print("\t\t  please enter your grade in recitation 1: ");
            double recitation1 = scan.nextDouble();

            System.out.print("\t\t  please enter your grade in recitation 2: ");
            double recitation2 = scan.nextDouble();

            /*Grades on recitation */
            System.out.print("\t\t  please enter your grade in project 1: ");
            double project1 = scan.nextDouble();

            System.out.print("\t\t  please enter your grade in project 2: ");
            double project2= scan.nextDouble();

            /*Grades on exam */
            System.out.print("\t\t  please enter your grade in exam: ");
            double exam_rating = scan.nextDouble();


            scan.close();
            // averages
            double quiz_avg = (quiz1 + quiz2) / 2;
            double recitation_avg = (recitation1 + recitation2)/2;
            double project_avg = (project1 + project2) / 2;

            // grade standimg
            double class_standing = ((quiz_avg) + (recitation_avg) + (project_avg)) / 3;
            double prelim_grade = ((class_standing * 2) + exam_rating) / 3;
            
            // Display the grade
            System.out.println("\n\t\t----------------------------------------------");
            System.out.printf("\t\t%s Name : %s\t\tCourse : %s", "", name, course);

            // display the grades
            System.out.printf("\n\n\t\t-- Average --\n\t\tQuiz Average : %.2f\n\t\tRecitation Average: %.2f\n\t\tProject Average: %.2f\n",
            quiz_avg, recitation_avg, project_avg);
            System.out.printf("\n\t\tClass standing: %.2f\n\t\tPrelim grade: %.2f\n", class_standing, prelim_grade);

            System.out.printf("\t\t\t\t\t\t%13s\n", "");
            System.out.println("\t\t----------------------------------------------");
        }   
    }
    

File: cs/act3/act3a.cs

/******** Grades Calculator ********
 * 
 * This program calculates the average of 3 grades
 * and displays the result.
 * 
 * *********************************/

using System;

class gradesA
{
    public static void Main(string[] args)
    {
        Console.WriteLine("=======================");
        Console.WriteLine("====== Grades Calculator =======");
        Console.WriteLine("=======================");

        string name, course;
        double q1, q2, rec1, rec2, proj1, proj2, exam_rating;
        double quiz_avg, rec_avg, proj_avg, class_standing, prelim_grade;

        //input
        Console.Write("Enter your name: ");
        name = Console.ReadLine();
        Console.Write("Enter your course: ");
        course = Console.ReadLine();
        Console.Write("Enter your quiz 1 grade: ");
        q1 = double.Parse(Console.ReadLine());
        Console.Write("Enter your quiz 2 grade: ");
        q2 = double.Parse(Console.ReadLine());
        Console.Write("Enter your recitation 1 grade: ");
        rec1 = double.Parse(Console.ReadLine());
        Console.Write("Enter your recitation 2 grade: ");
        rec2 = double.Parse(Console.ReadLine());
        Console.Write("Enter your project 1 grade: ");
        proj1 = double.Parse(Console.ReadLine());
        Console.Write("Enter your project 2 grade: ");
        proj2 = double.Parse(Console.ReadLine());
        Console.Write("Enter your exam rating: ");
        exam_rating = double.Parse(Console.ReadLine());

        // calculate the avg's
        quiz_avg = (q1 + q2) / 2;
        rec_avg = (rec1 + rec2) / 2;
        proj_avg = (proj1 + proj2) / 2;
        class_standing = (quiz_avg + rec_avg + proj_avg) / 4;
        prelim_grade = ((class_standing * 2) + (exam_rating)) / 3;

        //display the grade
        Console.WriteLine("---------------------");
        Console.Write($"\t\tName : {name}\t\tCourse : {course}\n");
        Console.Write($"\t\t-- Average -\n\t\tQuiz Average: {quiz_avg:F2}\n\t\tRecitation Average: {rec_avg:F2}\n\t\tProject Average: {proj_avg:F2}\n");
        Console.Write($"\n\t\tClass Standing: {class_standing:F2}\n\t\tPrelim Grade: {prelim_grade:F2}\n");

    }

}